<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/media/media/5.png</normaloff>:/images/media/media/5.png</iconset>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: red;
    font-size: 40px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="startGameButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>600</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: #ced1d8;
    border-style: outset;
    border-radius: 10px;
    font-size: 18px;
}
QPushButton:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}
QPushButton:hover {
    background-color: rgba(198, 198, 220, 0.6);
}</string>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seQuenceButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>650</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: #ced1d8;
    border-style: outset;
    border-radius: 10px;
    font-size: 18px;
}
QPushButton:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}
QPushButton:hover {
    background-color: rgba(198, 198, 220, 0.6);
}</string>
   </property>
   <property name="text">
    <string>排行榜</string>
   </property>
  </widget>
  <widget class="QLabel" name="topImageLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>531</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
